lms/djangoapps/api_manager/users/views.py

```diff
@@ -143,7 +143,10 @@ def _manage_role(course_descriptor, user, role, action):
     Helper method for managing course/forum roles
     """
     supported_roles = ('instructor', 'staff', 'observer', 'assistant')
-    forum_moderator_roles = ('instructor', 'staff', 'assistant')
+    role_perm_map = {
+        FORUM_ROLE_MODERATOR: ('instructor', 'staff'),
+        FORUM_ROLE_COMMUNITY_TA: ('assistant',),
```

Comment:
@antoviaque Took me a while to find an answer to this, on what privileges the TAs have, but check this out! :) https://github.com/edx-solutions/edx-platform/blob/4af2249a9d960b1316e5d7bde1a20d1765c8f000/common/djangoapps/django_comment_common/utils.py#L47

Marking them as Administrators would have the same problem, by the way-- the staff label is used if a person is a moderator or an administrator. See https://github.com/edx-solutions/edx-platform/blob/8e38a198b106bcf9586caca1a6766dcf14228a43/common/static/coffee/src/discussion/utils.coffee#L45